sycli/src/main.rs

```diff
@@ -665,3 +667,120 @@ fn main() {
         _ => {}
     }
 }
+
+/// Parse search criteria out of a filter string
+fn parse_filter(searches: &str) -> Vec<Criterion> {
+    use rpc::criterion::{Operation, Value};
+
+    let mut criterion = Vec::new();
+
+    // possible delimiters
+    let delimiters = vec![">=", "<=", "==", "!=", "<", ">", ":"];
+
+    'search: for search in searches.split_whitespace() {
```

Comment:
I wonder if there's a convenient crate to do more advanced tokenization. receptor supports things like name:"spaced name", but it seems like this wouldn't work so well here. If it's very inconvenient to find a good looking library for this though I think this is good enough for now.